README.md

```diff
@@ -93,18 +92,36 @@ jobs:
       - uses: actions/checkout@v4
       - uses: graalvm/setup-graalvm@v1
         with:
-          java-version: '23-ea' # or 'latest-ea' for the latest Java version available
+          java-version: '24-ea' # or 'latest-ea' for the latest Java version available
           distribution: 'graalvm'
           github-token: ${{ secrets.GITHUB_TOKEN }}
 ```
 
-</details>
-
 <details>
-<summary><h4>Template for older GraalVM releases</h4></summary>
+<summary><h4>Template for Oracle GraalVM via GraalVM Download Service</h4></summary>
+
+#### Prerequisites
+
+1. Obtain a token for the GraalVM Download Service. For this, replace `your@email.com` with your email address and run the following `curl` command:
+
+```bash
+curl -sS -X POST "https://gds.oracle.com/api/20220101/licenseAcceptance" \
+  -H "Content-Type: application/json" \
+  -d "{ \"email\": \"your@email.com\", \"licenseId\": \"D53FA58D12817B3CE0530F15000A74CA\", \"type\": \"GENERATE_TOKEN_AND_ACCEPT_LICENSE\"}"
+```
+
+The response should look like this:
+
+```json
+{"token":"<your-token>","status":"UNVERIFIED"}
+```
```

Comment:
Here for step 1, and step 4 below, you can indent this text by 4 spaces and then it will be rendered indented below the bullet point. That might make it a bit easier to read.

Same for the "Template for GraalVM Enterprise Edition" section below.

Reply:
Good suggestion, thanks!